go.mod

```diff
@@ -6,3 +6,5 @@ require (
 	github.com/pkg/diff v0.0.0-20210226163009-20ebb0f2a09e
 	github.com/rogpeppe/go-internal v1.8.1
 )
+
+require golang.org/x/exp v0.0.0-20220218215828-6cf2b201936e // indirect
```

Comment:
Does go mod tidy get rid of the "indirect" comment?

Reply:
Let's see. :)